Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Armoury.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class Armoury : MonoBehaviour
+{
+    public Player player;
+    public PointsSystem currency;
+
+    IList<int> GetBuyableUnits; //Change to UnitType once Units are in
+    IList<int> GetBuyableCards;
+    bool b_IsCastlePieceBuyable;
+
+    IDictionary<int, int> UnitPrices { get; set; } //Change key to UnitType once Units are in
+    IDictionary<int, int> CardPrices { get; set; }
+    IDictionary<int, int> CastlePiecePrices { get; set; }
```

Comment:
These need to be initialised somwhere. Probably in a public void Initialise() {} method.